packages/tasks/src/model-libraries.ts

```diff
@@ -1169,6 +1169,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path:"models/default.zip"`,
 	},
+	supertonic: {
+		prettyLabel: "Supertonic",
+		repoName: "supertonic",
```

Comment:
```suggestion
		repoName: "Supertonic",
```

(let's use the same for consistency)